doc/source/faq.rst

```diff
@@ -110,78 +110,6 @@ details.
 Visualizing Data in Qt applications
```

Comment:
So remove the entire thing about Qt applications?  I don't fully understand the second part of your question.